builds/service/service-deployment.yaml

```diff
@@ -30,7 +30,7 @@ steps:
     $testDir = '$(Build.SourcesDirectory)/test/Microsoft.Azure.Devices.Edge.Test'
     dotnet build $testDir
 
-    $binDir = Convert-Path "$testDir/bin/Debug/netcoreapp2.1"
+    $binDir = Convert-Path "$testDir/bin/Debug/netcoreapp3.1"
```

Comment:
We never run this so I think it is best to remove it altogether. We don't have to do it in this PR though.